src/exchanges/gdax/GDAXExchangeAPI.ts

```diff
@@ -199,15 +198,13 @@ export class GDAXExchangeAPI implements PublicExchangeAPI, AuthenticatedExchange
             return Promise.reject(new GTTError('No authentication details were given for this API'));
         }
         let gdaxOrder: OrderParams;
-        assert(order.side === 'buy' || order.side === 'sell');
```

Comment:
Why was the runtime check removed?

Reply:
@fb55 For consistency with the rest of the code, there are no runtime checks anywhere. Plus, PlaceOrderMessage above change's the 'side' property from a string to a Side, so there should be a compile time check.

Reply:
Sg 👍